x-pack/platform/plugins/shared/triggers_actions_ui/public/application/lib/action_connector_api/delete.ts

```diff
@@ -16,17 +16,20 @@ export async function deleteActions({
 }): Promise<{ successes: string[]; errors: string[] }> {
   const successes: string[] = [];
   const errors: string[] = [];
-  await Promise.all(
+
+  const results = await Promise.allSettled(
```

Comment:
Changed to `allSettled` to ensure that, when multiple ids are being processed, the success count comes back correctly. Prior to this, with multiple deletions, if one failed, the successful call would not be returned correctly.

Would be great to get extra eyes on this from someone from the response-ops team to ensure there isn't some extra context I'm missing. From what I can tell this is only used in the connectors list.